hack/build/init-buildx.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
```

Comment:
this script is a hacky experiment coped from the kind repo, I'm not ready to encourage replicating it throughout the project yet.